autogen/agentchat/chat.py

```diff
@@ -138,19 +138,19 @@ def initiate_chats(chat_queue: List[Dict[str, Any]]) -> List[ChatResult]:
         chat_queue (List[Dict]): a list of dictionaries containing the information about the chats.
 
         Each dictionary should contain the input arguments for `ConversableAgent.initiate_chat`. For example:
```

Comment:
Add a link to ConversableAgent.initiate_chat?

Reply:
yes